README.md

```diff
@@ -78,6 +80,7 @@ Fields:
  - `mediaCongrats` Type: String. Optional. Media used in dialog congratulation student on acceptance. (To do: see `dateAccepted`.)
  - `mediaMascot` Type: String. Optional. Media of the instutions mascot.
  - `messagingServiceSid` Type: String. Optional. A string used by Twilio to identify the messaging service of the institution.
+ - `messagingType` Type: \[String\].  Optional. The type of messaging service used by the bot: SMS, FB, Whatsapp, etc.
```

Comment:
We (and Twilio) use the term "messaging service" for something different.  I've been trying to get people to use the term "channel" for what you're talking about here. That's what Twilio use, and I think it's the closest thing to an industry standard term.

Reply:
Well, I'm not quite willing to change the name of the field, but I changed the description.  Is that good enough?